### ISSUE TITLE
Basic web app to serve the validator function online

### ISSUE THREAD

**Issue by bleonar5** [psych-ds/psychds-validator#20]
# TL;DR
Once the node.js validator is ready and complete, we will need to set up a basic web app that can import the validator using [browserify](https://browserify.org/). It will be a single static webpage with minimal UI elements. The stated plan is to use a react-based node application, but even that may be more bells and whistles than we need. Eventually, the decision will either come down to some notion of either simplicity or parity with existing system elements.

possible extensions of this include:
- embedding the [CEDAR wizard](https://cedar.metadatacenter.org/) directly
- creating some kind of landing page that points to documentation, tutorials, helper tools, and the validator itself

Logistical details like domain acquisition, hosting, etc will need to be worked out later on.
